packages/common/package.json

```diff
@@ -45,6 +45,11 @@
   },
   "dependencies": {
```

Comment:
We're missing `configu` store here that should also be inbuilt

Reply:
fixed/